src/fileseq/utils.py

```diff
@@ -119,18 +234,42 @@ def unique(seen, *iterables):
     return (i for i in chain(*iterables) if i not in seen and not _add(i))
 
 
-def pad(number, width=0):
+def pad(number, integral_width=0, decimal_places=None):
```

Comment:
Changing a keyword param of this public function could be considered a breaking change. Can we preserve the original name?